paper/paper.tex

```diff
@@ -50,11 +50,14 @@ \subsection{The enclose API}
 \end{figure}
 
 In \emph{\RE}, the \texttt{solver} is an instance of a struct that must be a subtype of \texttt{AbstractEnclosureAlgorithm}.
-If a user wants to add a new solver, they just have to add a new struct, say, \texttt{MyEnclosure} and implement the method \texttt{\_enclose}, which is automatically called by \texttt{enclose}.
+If a user wants to add a new solver, they just have to add a new struct, say, \texttt{MyEnclosure} and extend the method \texttt{enclose}, as the following code snippet demonstrates.
 %
 \begin{lstlisting}[language=Julia]
-_enclose(solver::MyEnclosure, f::Function,
-         D::Union{Interval, IntervalBox}; kwargs...)
+function enclose(f::Function,
```

Comment:
The following is necessary to not steal the `enclose` function.
```suggestion
function RangeEnclosures.enclose(f::Function,
```

Reply:
doesnt this mess indentation of the following lines?

Reply:
True. But we cannot indent them all by that much space in the paper. Either we indent them by a fixed amount or we use the `import` version. Then I prefer the second option, but it leads to a suboptimal page break. I propose to sneak in some `\vspace`s. I have to make a new comment for that...

Reply:
Does not work over deleted lines...
Here is what I suggest:

```latex
\vspace*{-2mm}
\begin{lstlisting}[language=Julia]
import RangeEnclosures: enclose
function enclose(f::Function,
                 dom::Union{Interval,IntervalBox},
                 solver::MyEnclosure; kwargs...)
    # solver-specific implementation
end
\end{lstlisting}
\vspace*{-1mm}
```

Reply:
> But we cannot indent them all by that much space in the paper.

that's because it would go over the listing margin, right?

Your suggestion looks good to me

Reply:
although if we removed the `Union{Interval,IntervalBox}` annotation (see previous comment), would we be able to indent that much?

Reply:
No, just tried it. 
![screenshot](https://github.com/JuliaReach/RangeEnclosures.jl/assets/9656686/7f477540-0737-4d80-90c4-41681b630c3b)

Reply:
`kwargs...` could (and should) go on its own line. But than it would be one `;` out.... damn so close

Reply:
maybe let's go with the following

```latex
\vspace*{-2mm}
\begin{lstlisting}[language=Julia]
import RangeEnclosures: enclose
import IntervalArithmetic: Interval
function enclose(f::Function,
                 dom::Union{Interval,IntervalBox},
                 solver::MyEnclosure; kwargs...)
    # solver-specific implementation
end
\end{lstlisting}
\vspace*{-1mm}
```

Reply:
I had to add more negative space because the page break would be worse. But I added this suggestion in a new commit. So I close the discussion for now.